aci/resource_aci_l3extlifp.go

```diff
@@ -61,12 +63,22 @@ func resourceAciLogicalInterfaceProfile() *schema.Resource {
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
```

Comment:
object also allows for annotation, should we add this to schema?

Reply:
I don't think we have done it for other relationship attributes?

Reply:
ok

Reply:
Discussed internally